aquadoggo/src/rpc/methods/publish_entry.rs

```diff
@@ -42,16 +45,32 @@ pub async fn publish_entry(
     let entry = decode_entry(&params.entry_encoded, Some(&params.message_encoded))?;
     let message = Message::from(&params.message_encoded);
 
-    // Retreive author and schema
     let author = params.entry_encoded.author();
-    let schema = message.schema();
-
-    // Determine log_id for author's schema
-    let schema_log_id = Log::get(&pool, &author, &schema).await?;
 
-    // Check if log_id is the same as the previously claimed one (when given)
-    if schema_log_id.is_some() && schema_log_id.as_ref() != Some(entry.log_id()) {
-        Err(PublishEntryError::InvalidLogId)?;
+    // Determine expected log id for new entry: a `CREATE` entry is always stored in the next free
+    // user log. An `UPDATE` or `DELETE` message is always stored in the same log that its original
+    // `CREATE` message was stored in.
```

Comment:
This is only true in a single writer situation, right? Just commenting to check the scope we're talking in here ;-p

Reply:
Yes, as we haven't introduced any multi-writer mechanics anywhere else yet this PR assumes a single-writer scenario.